Expensify/App
# Hide the composer when message is edited

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7265

### Tests | QA Steps
1. Open app
2. Navigate to any chat
3. Send messages
4. Hit Edit to make any changes



### Tested On


### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/152863788-e389d2d2-56ff-47a2-a7d8-c7063debbfa3.mp4


#### Mobile Web


#### 


#### iOS


#### Android
